pkg/internal/ebpf/gosql/gosql.go

```diff
@@ -82,6 +82,10 @@ func (p *Tracer) UProbes() map[string]map[string]ebpfcommon.FunctionPrograms {
 	return nil
 }
 
+func (p *Tracer) Tracepoints() map[string]ebpfcommon.FunctionPrograms {
```

Comment:
I added support for tracepoints too. I had forgotten that the old helper I used was reading kernel memory only and it was useful to confirm what the addresses of the values were with a tracepoint. I've had to do this a few times in the past, so I keep adding and removing tracepoint support. I decided to add it and keep it for now. We have no tracepoints yet, but it might be useful again in the future.

We also have this outstanding issue where kretprobes can be unreliable, so maybe switching to a tracepoint exit on those would be better.